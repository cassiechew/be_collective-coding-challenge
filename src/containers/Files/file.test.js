import { render, screen } from '@testing-library/react';
import { create } from 'react-test-renderer';
import React from 'react';
import File from './file';

describe('File component', () => {
    test('contains name text', () => {
        render(<File name="test" />);
        const text = screen.getByText(/test/i);
        expect(text).toBeInTheDocument();
    });

    test('contains size text', () => {
        render(<File size="size" />);
        const text = screen.getByText(/size/i);
        expect(text).toBeInTheDocument();
    });
});

describe('snapshot test', () => {
    test('testing footer', () => {
      let tree = create(<File name="name" size="size" />);
      expect(tree.toJSON()).toMatchSnapshot();
    });
});