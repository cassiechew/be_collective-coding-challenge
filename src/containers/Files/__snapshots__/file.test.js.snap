// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test testing footer 1`] = `
<div
  className="container__file"
>
  <div>
    <img
      alt="failed"
      src="file.png"
    />
     
    name size
  </div>
</div>
`;
